src/components/ebay-progress-spinner/index.marko

```diff
@@ -0,0 +1,2 @@
+$ var label = input.a11yText ? input.a11yText : "Busy"
+<span ...input class=[ "spinner", input.size === "large" && "spinner--large"] aria-label=label role="img"></span>
```

Comment:
For `class`, attribute, allow the users to override it by adding:
`input.class` in the array.
Also you should